src/components/Error/Errors.js

```diff
@@ -1,17 +1,41 @@
 import React from 'react'
 import styled from 'react-emotion'
 
+import WarningImage from '../../assets/warning.svg'
+
 const ErrorContainer = styled('div')`
-  color: white;
-  padding: 20px;
+  color: black;
+  padding: 40px 20px;
   text-align: center;
-  background: red;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  max-width: 400px;
+  margin: 0 auto 0;
+`
+
+const Warning = styled('img')`
+  width: 40px;
 `
 
+const H2 = styled('h2')`
+  font-family: Overpass;
+  font-weight: 300;
+  font-size: 28px;
+  color: #2b2b2b;
+  text-align: center;
+`
+
+const Message = styled('div')``
+
 export const NetworkError = ({ message }) => (
   <ErrorContainer>
-    {message}
-    <br />
-    Please change your dapp browser to Mainnet or Ropsten
+    <Warning src={WarningImage} />
+    <Message>
+      <H2>{message}</H2>
+      Change your network to the Main Ethereum Network or Ropsten Test Network
```

Comment:
Rinkeby is actually supported (for `.test`) https://docs.ens.domains/en/latest/introduction.html#ens-on-ethereum . It may start working if you merge the latest dev as I added the support at [this commit](https://github.com/ensdomains/ens-app/commit/9010157d11bdbc8f6064a5a250406e1f2684f294) but you may want to change wording to add Rinkeby as well as Goerli (then only network not supported is Kovan and non ganache based private network).

Reply:
1. I can still see Kovan network as it is same network id as Gorli (probably impossible to programatically determine the difference unless you hard cord contract id). Is this expected behavior?

<img width="271" alt="screenshot 2019-02-14 at 05 23 32" src="https://user-images.githubusercontent.com/2630/52786695-cb51e100-3018-11e9-8c86-0c5757129dd2.png">

2. I added POA network (https://core.poa.network) as a custom RPC end point but that simply shows as "Private Network"

<img width="1312" alt="screenshot 2019-02-14 at 05 24 52" src="https://user-images.githubusercontent.com/2630/52786908-72cf1380-3019-11e9-98e2-64d202cfee85.png">

Basically I can't test the case where it shows the warning message